src/components/views/login/PasswordLogin.js

```diff
@@ -96,10 +96,12 @@ module.exports = React.createClass({displayName: 'PasswordLogin',
             <div>
                 <form onSubmit={this.onSubmitForm}>
                 <input className="mx_Login_field" type="text"
+                    name="username" // make it a little easier for browser's remember-password
```

Comment:
Note this will mean anyone why's already told chrome to remember their passwords will have to re-enter their credentials. We could remove this if we're not happy with that: I figure it's better to have them and if so we'd better do it sooner rather than later.